### PR TITLE
Aa/add meta tags

### PR DESCRIPTION
**Pull request by Rajp123**
Aa/add meta tags

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+    <meta name="keywords" content="data science, software development, jobs">
+    <meta name="author" content="Jovian">
+    <meta name="robots" content="index, follow">
+    <meta property="og:title" content="Jovian Careers">
+    <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta property="og:image" content="/metaimage.png">
+    <meta name="twitter:title" content="Jovian Careers">
+    <meta name="twitter:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="twitter:image" content="/metaimage.png">
+    <title>FERDI SAHIN </title>
```
Comment (Rajparu2253) [inline src/index.html:15]: make the title bigger

Comment (Rajparu2253) [inline src/index.html:15]: make title longer than this

File: src/index.html [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
 <head>
-  <title>FERDI SAHIN </title>
+  <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="keywords" content="data science, software development, jobs">
+    <meta name="author" content="Jovian">
+    <meta name="robots" content="index, follow">
+    <meta property="og:title" content="Jovian Careers">
+    <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
```
Comment (Rajparu2253) [inline src/index.html:10]: make description smaller

File: src/index.html (hunk 1 above)
Comment (Rajp123) [reply src/index.html:15]: done

**Pull request merged**